apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/command/PageContentCommand.java

```diff
@@ -15,211 +15,219 @@ public sealed interface PageContentCommand {
         Instant requestedOn();
 
         public record AddLineSeparator(
-            WaTemplate page,
-            Integer orderNumber,
-            long userId,
-            String adminComments,
-            Instant requestedOn) implements PageContentCommand {
+                        WaTemplate page,
+                        Integer orderNumber,
+                        long userId,
+                        String adminComments,
+                        Instant requestedOn) implements PageContentCommand {
```

Comment:
Suggestion: Formatting seems a bit much here. Ensure you are setting tab size to 2 spaces.